.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ version: 2
 jobs:
   generate-bindings:
     docker:
-      - image: circleci/openjdk:8u171-jdk-node-browsers
```

Comment:
most of these changes come from the cherry-picked https://github.com/palantir/conjure-python-client/pull/136/commits/a79e66a8a179651a04e42485fe23603a9125c980 which is from develop